app/views/layouts/internal.html.erb

```diff
@@ -40,7 +40,7 @@
 
   <div class="container mt-3">
     <% flash.each do |type, message| %>
-      <div class="alert alert-<%= type == "notice" ? "success" : "danger" %>">
+      <div class="alert alert-<%= type == "notice" || type == "success" ? "success" : "danger" %>">
```

Comment:
This is more of a bandaid fix for now. I'll make a separate PR to use only `success` or `notice`.